src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -91,6 +91,18 @@ case class MethodConfigurationNamePair(
                    destination: MethodConfigurationName
                    )
 
+@ApiModel(value = "Entity copy definition")
+case class EntityCopyDefinition(
+                   @(ApiModelProperty@field)(required = true, value = "The source workspace of the entities")
+                   sourceWorkspace: WorkspaceName,
+                   @(ApiModelProperty@field)(required = true, value = "The destination workspace for the entities")
+                   destinationWorkspace: WorkspaceName,
```

Comment:
General API question - do we need to specify the destination workspace explicitly? Or will we always be copying into the "current" workspace?